application/F3DStarter.cxx

```diff
@@ -320,10 +321,12 @@ int F3DStarter::Start(int argc, char** argv)
       }
     }
 
+    char* noDataForceRender = std::getenv("CTEST_F3D_NO_DATA_FORCE_RENDER");
```

Comment:
```suggestion
    bool noDataForceRender = vtksys::SystemTools::HasEnv("CTEST_F3D_NO_DATA_FORCE_RENDER");
```

Reply:
nope, no access to vtksys in application

Reply:
ok